website/src/IndexPage.tsx

```diff
@@ -404,8 +404,7 @@ export function SupportedNetworks() {
             )
           })}
         </ul>
-        <Text.P14 color="White64" sx={{ mt: '32px' }}>
-          *{t('index.supportedNetworks.betaWarning')}
+          <Text.P14 color="White64" sx={{ mt: '32px' }}>
         </Text.P14>
```

Comment:
Actually can you remove the whole `<Text.P14 color="White64" sx={{ mt: '32px' }}></Text.P14>` (now empty) component please? I would make a suggestion that you can just apply here but it seems I can't due to how the diff looks.

Reply:
Sure thign!